cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TransactionsCommandTest.scala

```diff
@@ -411,5 +415,107 @@ All users may terminate their own currently executing transactions.
         }
       }
     }
+    section("Combining transaction commands", id="query-combine-tx-commands") {
+      p(
+        """
+In difference to other show commands, the show and terminate transactions commands can be combined in the same query.
```

Comment:
possibly add code format to SHOW and TERMINATE TRANSACTIONS?

Reply:
```suggestion
In difference to other show commands, the `SHOW` and `TERMINATE TRANSACTIONS` commands can be combined in the same query.
```

like this you mean?

Reply:
yes